Skewer adaptation

nluaces opened the pull request:
Updated README file using the Skewer project.

nluaces committed:
Add Skewer submodule

nluaces committed:
Add new README file generated by Skewer.

ssorj commented on .gitmodules:
```diff
@@ -0,0 +1,3 @@
+[submodule "skewer"]
```
I think we should consider using git-subrepo instead.  The main reason is simply that it avoids one extra step for the user.  But let me know if you tried it and prefer submodules.

ssorj replied:
The subrepo-based instructions: https://github.com/skupperproject/skewer#setting-up-skewer-for-your-own-example and https://github.com/skupperproject/skewer#setting-up-skewer-for-your-own-example

nluaces replied:
Changed to git-subrepo, thanks for the instructions!

ssorj commented on skewer.yaml:
```diff
@@ -0,0 +1,141 @@
+        - run: "INSERT INTO product VALUES(DEFAULT, 'Orange', '4056');"
+    postamble: |
+      From any cluster, access the `product` tables in the `markets` database to view contents.
+summary: |
```
The summary is rendered at the end, so this text should either go back to the start (the overview) or be recast as "what you did", not what you will do.

nluaces replied:
Thanks! Summary updated.

nluaces committed:
Removed submodule

nluaces committed:
git subrepo clone https://github.com/skupperproject/skewer subrepos/skewer

subrepo:
  subdir:   "subrepos/skewer"
  merged:   "3c2a955"
upstream:
  origin:   "https://github.com/skupperproject/skewer"
  branch:   "main"
  commit:   "3c2a955"
git-subrepo:
  version:  "0.4.3"
  origin:   "https://github.com/ingydotnet/git-subrepo"
  commit:   "2f68596"

nluaces committed:
Updated summary
